include/boost/system/boost_system.natvis

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+<AutoVisualizer xmlns="http://schemas.microsoft.com/vstudio/debugger/natvis/2010">
+
+    <Type Name="boost::source_location">
+        <DisplayString>{function_,sb} @ {file_,sb}:{line_}</DisplayString>
+        <Expand>
+            <Item Name="file">file_,sb</Item>
+            <Item Name="func">function_,sb</Item>
+        </Expand>
+    </Type>
+
+    <Type Name="boost::system::error_category">
+        <DisplayString Condition="id_ == 0xb2ab117a257edfd0">[generic_category]</DisplayString>
+        <DisplayString Condition="id_ == 0xb2ab117a257edfd1">[system_category]</DisplayString>
+        <DisplayString Condition="id_ == 0xb2ab117a257edfd2">[interop_category]</DisplayString>
+        <DisplayString>error_category@{(size_t)this}</DisplayString>
```

Comment:
There shouldn't be a `size_t` cast here. The identifier of a category with `id_ == 0` is the address of `this`.

And of course when `id_ != 0`, the category identifier should be displayed, not the address.